dluhc-component-library/src/api/api.ts

```diff
@@ -0,0 +1,12 @@
+import WKT from "ol/format/WKT.js";
+import { Geometry } from "ol/geom";
+
+export async function fetchDataset(geometry: Geometry) {
+  const format = new WKT();
+  const polygon = format.writeGeometry(geometry);
+
+  const promise = await fetch(
```

Comment:
`Non-blocker` I probably should have spotted this on the first review but instead of sabing the promise as a variable then calling json we should probably just chain it.

```
return await fetch(
  `https://www.planning.data.gov.uk/entity.geojson?limit=100&geometry=${polygon}`,
).json();

Reply:
Due to the fetch returning a promise it looks a bit more like 
`  return await (
    await fetch(
      `https://www.planning.data.gov.uk/entity.geojson?limit=100&geometry=${polygon}`,
    )
  ).json();`

which is a bit icky but happy to make the change if you are

Reply:
Doing `.then((res) => res.json())` after the initial fetch might work